lib/local-links-manager/import/csv_downloader.rb

```diff
@@ -26,4 +33,10 @@ def downloaded_csv
 
     response.body
   end
+
+  def name_converter(conversions)
```

Comment:
yeah I struggled with finding a good name. Went with `field_name_converter` in the end, as that seems to match CSV terminology somewhat (without using the word 'header' again)?